client/packages/common/src/hooks/useNativeClient/types.ts

```diff
@@ -66,3 +68,5 @@ export type BarcodeScanner = {
   product?: string;
   connected: boolean;
 };
+
+export type ScannerType = 'usb_serial' | 'usb_keyboard';
```

Comment:
used string constants, since it's slightly easier to use when only a few variations (it's still just as much type safety)